libs/test-setup/src/test_api_args.rs

```diff
@@ -201,7 +199,7 @@ impl TestApiArgs {
         self.db.provider
     }
 
-    pub fn shadow_database_url(&self) -> Option<&'static str> {
+    pub fn shadow_database_url(&self) -> Option<&str> {
```

Comment:
can also now be reverted but also doesn't hurt to leave it, there's no need for it to be static